.eslintrc.json

```diff
@@ -0,0 +1,26 @@
+{

```

Comment:
## Your Readme :memo:
![image](https://user-images.githubusercontent.com/93039359/199366863-3515c4ec-7132-4ca3-a6c1-e15ba3faf0a3.png)


- [ ] Your readme is not descriptive enough Kindly make it descriptive using microverse [template](https://github.com/microverseinc/readme-template/blob/master/README.md) in your project :+1: